docs/src/components/App/Header/headerUtils.js

```diff
@@ -0,0 +1,48 @@
+import { pipe } from 'lodash/fp';
+
+export const buildRoutes = inputRoutes =>
+  inputRoutes.reduce((acc, route) => {
+    const category = route.category || 'Other';
+    const initialArrayVal = acc[category] || [];
+    return {
+      ...acc,
+      [category]: [...initialArrayVal, route]
+    };
+  }, {});
+
+export const sortRoutes = (routeA, routeB) => {
```

Comment:
I would suggest renaming this to `compareRoutes`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Syntax